<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TMOHDDListeFormu</class>
 <widget class="QDialog" name="TMOHDDListeFormu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HDD Liste Formu</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../Kaynaklar.qrc">
    <normaloff>:/resimler/hdd.jpg</normaloff>
    <normalon>:/resimler/hdd.jpg</normalon>:/resimler/hdd.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">TMOHDDListeFormu{

	
	background-image: url(:/resimler/hdd.jpg);
}

QLabel {
   font:Bold; 
   color: #ffffff;
}
QGroupBox {
    font:Bold;
color: #ffffff;
}
QPushButton{font:Bold;}
QRadioButton{
    font:Bold;
color: #ffffff;
}url(:/resimler/hdd.jpg)</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0">
    <widget class="QGroupBox" name="GPAramaBolumu">
     <property name="title">
      <string>HDD ARAMA BÖLÜMÜ</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="1">
       <widget class="QGroupBox" name="GPParcaModeli">
        <property name="title">
         <string>Parça Boyutu</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="LPModeliDeger">
           <property name="text">
            <string>Değer:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="LEPModeliDeger"/>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QGroupBox" name="GPPModeliFiltreTuru">
           <property name="title">
            <string>Filtre Türü</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QRadioButton" name="RBPModeliFTIleBaslayan">
              <property name="text">
               <string>İle Başlayan</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QRadioButton" name="RBPModeliIFTIleBiten">
              <property name="text">
               <string>İle Biten</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QRadioButton" name="RBPModeliFTIceren">
              <property name="text">
               <string>İçeren</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="PBAra">
        <property name="text">
         <string>ARA</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QGroupBox" name="GPParcaMarkasi">
        <property name="title">
         <string>Parça Markası</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="LPMarkasiDeger">
           <property name="text">
            <string>Değer:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="LEPMarkasiDeger"/>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QGroupBox" name="GPPMarkasiFiltreTuru">
           <property name="title">
            <string>Filtre Türü</string>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QRadioButton" name="RBPMarkasiFTIleBaslayan">
              <property name="text">
               <string>İle Başlayan</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QRadioButton" name="RBPMarkasiFTIleBiten">
              <property name="text">
               <string>İle Biten</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QRadioButton" name="RBPMarkasiFTIceren">
              <property name="text">
               <string>İçeren</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="PBKapat">
        <property name="text">
         <string>KAPAT</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTableWidget" name="TWTablo"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../Kaynaklar.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>PBAra</sender>
   <signal>clicked()</signal>
   <receiver>TMOHDDListeFormu</receiver>
   <slot>aramaYap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>849</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>878</x>
     <y>-14</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PBKapat</sender>
   <signal>clicked()</signal>
   <receiver>TMOHDDListeFormu</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>940</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>976</x>
     <y>-5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RBPMarkasiFTIleBaslayan</sender>
   <signal>clicked()</signal>
   <receiver>TMOHDDListeFormu</receiver>
   <slot>aramaYap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>-3</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RBPMarkasiFTIleBiten</sender>
   <signal>clicked()</signal>
   <receiver>TMOHDDListeFormu</receiver>
   <slot>aramaYap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RBPMarkasiFTIceren</sender>
   <signal>clicked()</signal>
   <receiver>TMOHDDListeFormu</receiver>
   <slot>aramaYap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>303</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RBPModeliFTIleBaslayan</sender>
   <signal>clicked()</signal>
   <receiver>TMOHDDListeFormu</receiver>
   <slot>aramaYap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>472</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>978</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RBPModeliIFTIleBiten</sender>
   <signal>clicked()</signal>
   <receiver>TMOHDDListeFormu</receiver>
   <slot>aramaYap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>586</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>979</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RBPModeliFTIceren</sender>
   <signal>clicked()</signal>
   <receiver>TMOHDDListeFormu</receiver>
   <slot>aramaYap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>703</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>979</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LEPModeliDeger</sender>
   <signal>textChanged(QString)</signal>
   <receiver>TMOHDDListeFormu</receiver>
   <slot>aramaYap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>526</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LEPMarkasiDeger</sender>
   <signal>textChanged(QString)</signal>
   <receiver>TMOHDDListeFormu</receiver>
   <slot>aramaYap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>261</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>35</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>aramaYap()</slot>
 </slots>
</ui>
